pkg/gadgettracermanager/gadgettracermanager.go

```diff
@@ -269,22 +268,16 @@ func NewServer(conf *Conf) (*GadgetTracerManager, error) {
 			opts = append(opts, containercollection.WithInitialKubernetesContainers(g.nodeName))
 		}
 	case "auto":
-		if runcfanotify.Supported() {
-			log.Infof("GadgetTracerManager: hook mode: fanotify (auto)")
-			opts = append(opts, containercollection.WithRuncFanotify())
-			opts = append(opts, containercollection.WithInitialKubernetesContainers(g.nodeName))
-		} else {
-			log.Infof("GadgetTracerManager: hook mode: podinformer (auto)")
-			opts = append(opts, containercollection.WithPodInformer(g.nodeName))
-			podInformerUsed = true
-		}
+		log.Infof("GadgetTracerManager: hook mode: fanotify (auto)")
+		opts = append(opts, containercollection.WithContainerFanotify())
+		opts = append(opts, containercollection.WithInitialKubernetesContainers(g.nodeName))
```

Comment:
Avoid changing this as it is changed in [hooks: add "fanotify+ebpf"](https://github.com/inspektor-gadget/inspektor-gadget/pull/1709/commits/8bff4ec623a955678e14dd5f82f4872c4e080c25).